src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -123,7 +138,13 @@ class SourcesTree extends Component {
 
     if (nextProps.sources.size === 0) {
       // remove all sources
-      this.setState(createTree(nextProps.sources, this.props.debuggeeUrl));
+      this.setState(
+        createTree(
+          nextProps.sources,
+          this.props.debuggeeUrl,
```

Comment:
since everything else here is nextProps... this might also need to be nextProps... is there any information on why we would want the old url?

Reply:
maybe we should be more explicit here. like giving an empty list, empty string, and empty root?